app/javascript/components/shared/ListingForm.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import Form from "./Form";
+import Input from "./FormInput";
+
+const ListingForm = props => {
+  return (
+    <Form {...props.form}>
+      <div className="row">
+        <input
+          type="text"
+          name="listing[created_by_id]"
+          id="listing_created_by_id"
+          value={props.form.creator_id}
+          hidden
+        />
+        <div className="col">
+          <label htmlFor="title">Listing Title</label>
+          <Input
+            name="listing[title]"
+            className="form-control"
+            id="title"
+            aria-describedby="nameHelp"
```

Comment:
Can you check the interface for `Input` `aria-describedby` and `className` aren't used in that component, so those props don't need to be here.